dvc/repo/status.py

```diff
@@ -113,4 +115,15 @@ def status(
             remote=remote,
             all_tags=all_tags,
         )
+
+    ignored = list(
+        compress(
+            ["--all-branches", "--all-tags", "--jobs", "--remote", "--cloud"],
```

Comment:
Why do we define `remote` and `cloud` here, if we check for them in previous step?

Reply:
Hmm, you are right

Reply:
Fixed